lib/src/polymer_base.dart

```diff
@@ -229,6 +229,12 @@ abstract class PolymerBase implements CustomElementProxyMixin {
         .apply([path, convertToJs(value), fromAbove], thisArg: jsElement);
   }
 
+  /// Notify that a value at a path has been changed (without passing the value).
+  void notifyPath1(String path) {
```

Comment:
100% agree with you.  

I think the solution to change the signature into `notifyPath(String path,[value,fromAbove])` should be the better choice.

I'll fix it in `polymer-dart` too and adding a changelog warning then.